docs/communication_protocols/dronecan_protocol.rst

```diff
@@ -47,7 +47,42 @@ uavcan.protocol.NodeStatus (Data Type ID = 341)
 The DroneCAN protocol requires that all nodes on a DroneCAN network periodically publish their status using the Node Status message. Vertiq modules support this behavior to conform 
 to the standard. A uavcan.protocol.NodeStatus message is published at 1 Hz during operation, reporting its health, current mode, and uptime.
 
-In normal operation a Vertiq module should always report its Health as OK, and its mode as Operational. No sub-modes or vendor specific status codes are currently supported.
+Vertiq's Node Health Indications
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Vertiq modules support all four standard DroneCAN `health indications <https://dronecan.github.io/Specification/7._List_of_standard_data_types/#:~:text=fatal%20malfunction.%0Auint2-,health,-%23%0A%23%20Current%20mode.%0A%23%0A%23%20Mode>`_. Modules enter each state given the following:
+
+Critical
+""""""""""
+#. The module's :ref:`total derate <derates>` reaches 0
+#. Your module's drive mode is in lockout due to :ref:`power safety <power_safety>`
+
+Warning
+""""""""""
+#. The module's supply voltage is greater than your drive's :ref:`v_max_start <brushless_drive>`. This typically indicates that the module is regenerating into a supply that cannot handle regenerated current, such as a typical benchtop power supply, and is limiting how much it regenerates to protect itself
+#. Your microcontroller's temperature is within 10°C of your module's :ref:`microcontroller temperature derate <microcontroller_derate>` window
+#. Your module's coil temperature is within 10°C of your module's :ref:`coil temperature derate <coil_temp_derate>` window
+
+Error
+""""""""
+#. The module is actively in :ref:`timeout <manual_timeout>`
+#. The module's :ref:`total derate <derates>` is less than 1 but greater than 0
+
+Healthy
+""""""""""
+When none of the conditions of Critical, Warning, or Error are met, the module is considered Healthy.
+
+
+Vertiq's Vendor Specific NodeStatus Code Breakdown
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Vertiq's vendor specific NodeStatus bits are used to indicate the state of various possible module errors. The bits are broken down as follows:
+
```

Comment:
Any chance you could give us an example of what it would look like in a certain condition to illustrate your point? Like, pick some interesting arbitrary state, maybe our microcontroller is derating and our voltage is too high or whatever, and show what  you would expect the status code to be. Doesn't have to be that exact situation, but some interesting example. People really benefit a lot from examples, even if it seems obvious